src/main/python/cfn_sphere/exceptions.py

```diff
@@ -3,7 +3,7 @@
 
 class CfnSphereException(Exception):
     def __init__(self, message="", boto_exception=None):
-        self.pretty_string = message
+        self.pretty_string = str(message)
```

Comment:
No: https://github.com/cfn-sphere/cfn-sphere/blob/master/src/main/python/cfn_sphere/stack_configuration/__init__.py#L34

In python 3 there is no `message` attribute in `Exception`, so you have to use `str()`